antidox/tester.py

```diff
@@ -27,6 +28,8 @@ def wiki_write(result, header):
 
 def log_change():
   """ gets latest revisions and cleans them """
+  dlp_response = perspective.dlp_request(dlp, apikey_data, text)
```

Comment:
Only the `get_client` call should be moved outside the loop. The `perspective.dlp_request()` call should stay inside the loop, since you need to call it for every change. (This code should result in an error if you try to run it.)